src/control/cmd/daos/flags.go

```diff
@@ -241,3 +241,34 @@ func (f *ContTypeFlag) UnmarshalFlag(fv string) error {
 	f.Set = true
 	return nil
 }
+
+type FsCheckFlag struct {
+	Set   bool
+	Flags C.uint64_t
+}
+
+func (f *FsCheckFlag) UnmarshalFlag(fv string) error {
+	if fv == "" {
+		return errors.New("empty filesystem check flags")
+	}
+
+	f.Flags = 0
```

Comment:
NB: Go automatically initializes stuff, no need to do it explicitly. https://go.dev/tour/basics/12